testplan/web_ui/testing/src/__tests__/documents/index.js

```diff
@@ -0,0 +1,11 @@
+if(process.env.NODE_ENV !== 'production') {
```

Comment:
Shall we keep it in production?

Reply:
Definitely not. I was debugging an unrelated issue in Chrome and saw that these giant JSON files are loaded into the browser  on every load. Pointless memory consumption.

However, this `.../__tests__/documents/` directory accounted for 52% of the lines changed (totaling 4,557 lines) in my initial squash commit, so this directory is now gone and the fake reports are back to normal.